.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.20.0rc1
+current_version = 0.20.0
```

Comment:
This will be bumped to `0.21.0b1` when we cut v0.20.0-b1 next week. What do you think about bumping this to `0.20.1a1` for the time being, on `develop` only? Silly and unnecessary?

Reply:
The only concern that I would have is that it might confuse people that there is an actual `0.21.0a1` release but if we never do alpha releases then I guess that would be less confusing

Reply:
not silly! I've had the same thought, we can determine if folks are installing directly from source this way as well.

Reply:
Alright! Let's do it!